packages/did/src/index.ts

```diff
@@ -11,5 +11,8 @@ export * from './DidDocumentExporter/index.js'
 export * from './DidResolver/index.js'
 export * as DidChain from './Did.chain.js'
 export * as DidUtils from './Did.utils.js'
-
+export * from './DemoKeystore/DemoKeystore.js'
```

Comment:
Why are these items exported again? I think `DemoKeystore` is already exported above, isn't it? Same goes for the others.

Reply:
fixed